docs/ci.md

```diff
@@ -0,0 +1,152 @@
+---
+title: With CI
+description: "Test that your images build and your servers start"
+layout: docs
+---
+
+Once you've bought into container-based development,
+you'll want to set up CI to ensure your servers don't break.
+
+We think of this in two pieces:
+
+1) Run Tilt in CI to deploy to a single-use cluster.
+
+2) Run a single-use Kubernetes in CI that you can immediately throw away.
```

Comment:
Hm, I'm not sure this gets across the relationship between the steps. Consider something like:
```
`tilt ci` is our recommended way to make sure your servers haven't broken from your latest change--it spins up your whole Tilt config and makes sure everything is healthy. To run this as part of your CI pipeline, you'll need a Kubernetes cluster to run it against--ideally a single-use cluster that can be easily thrown away when you're done.

Thus, we have two things to dig into here:
1. how to use `tilt ci` to make sure your servers are healthy
2. how to get access to a Kubernetes cluster from CI that you can run `tilt ci` against
```

Reply:
ya, i think this is a good criticism - rewrote it a bit